contributions/catalog/catalog_rest_service.py

```diff
@@ -21,33 +22,43 @@
     logging.basicConfig(datefmt='%Y-%m-%dT%H:%M:%S', format=log_format, level=logging.DEBUG)
 else:
     logging.basicConfig(datefmt='%Y-%m-%dT%H:%M:%S', format=log_format, level=logging.INFO)
-if cfg and cfg.URL_PREFIX:
-    prefix = cfg.URL_PREFIX
-    staticpath = prefix + '/static'
+
+URL_PREFIX = os.getenv("URL_PREFIX", "")
+if URL_PREFIX:
+    staticpath = URL_PREFIX + '/static'
 else:
     staticpath = '/static'
 
 template_dir = os.path.join(os.path.abspath('webapps'), 'templates')
 static_dir = os.path.join(os.path.abspath('webapps'), 'static')
 app = Flask(__name__, instance_relative_config=True, static_url_path=staticpath, static_folder=static_dir,
             template_folder=template_dir)
-app.config.from_object(cfg)
 
 init_app(app)
+CORS(app, resources={r"/localhost:5050/*": {"origins": "*"}}, supports_credentials=True)
 app.register_blueprint(contribute_bp)
 
```

Comment:
I am not very sure if we need to have CORS for the catalog but if we want to add, we probably want to add CORS_ENALBED variable for the catalog, like the one for the contributions. Also, if you want to allow every origin for your cors, you can simplify the code like

init_app(app)
app.register_blueprint(contribute_bp)
CORS(app.app)